### PR TITLE
Update ModReg schedule for AY21/22 Semester 1

### LINKED ISSUE #3292: Update ModReg Data for AY21/22 S1
Better late than never right?

Note: This is slightly time sensitive (i.e. if we do it too late, there is no point). Let's try to complete it before Select Tutorials / Labs Round 0.

# Steps

1. Follow https://github.com/nusmodifications/nusmods/pull/2764
2. Test it make sure it works by setting your system time
3. Update [MAINTENANCE.md](https://github.com/nusmodifications/nusmods/blob/master/MAINTENANCE.md) to include this step under "Every Semester"

If anybody wants to take this up, drop a comment below!

### PR DESCRIPTION
**Pull request by bnjmnt4n**
## Context

Information from:
- http://www.nus.edu.sg/ModReg/docs/UGCPE_ModRegSchedule.pdf
- http://www.nus.edu.sg/ModReg/docs/GD_ModRegSchedule.pdf
Correct as at 27 June 2021 7pm.

Closes #3292.

## Implementation

Updates the dates and documentation.

### CODE REVIEW

**Comment by bnjmnt4n** [comment]
Screenshot of the popup for tomorrow's select tutorials round 0:

image

File: MAINTENANCE.md [hunk 1]
```diff
@@ -15,7 +15,11 @@ Create a new issue on GitHub with this checklist after the finals every semester
 - [ ] Deploy! :tada: :tada:
 - [ ] Monitor Sentry and Messenger for issues
 
+Reference PRs: [PR #3286](https://github.com/nusmodifications/nusmods/pull/3286) and [PR #3287](https://github.com/nusmodifications/nusmods/pull/3287)
```
Comment (chrisgzf) [inline MAINTENANCE.md:18]: Nice! Thanks for adding reference PRs to the other maintenance tasks too!

**Review by chrisgzf — APPROVED** [review]
LGTM and congrats on your first contribution to NUSMods! 🎉

**Review by taneliang — APPROVED** [review]
Wow thanks for doing this @bnjmnt4n!

**Pull request merged**